md/guest-user.md

```diff
@@ -0,0 +1,59 @@
+# Guest user access
+
+It is possible to allow public access to some living applications (without the need to sign in to Bonita).  
+
+When accessing a public application without any active session on Bonita platform the user will be automatically logged in with a guest user account.
+
+:::info 
+**Note:** For Enterprise, Performance, Efficiency, and Teamwork editions only.
+:::
+
+::: warning  
+ This feature is different from the auto-login feature that was available in the 6.x versions of Bonita BPM as it do gives directly access to a process form publicly. It is intended for living applications access.
+:::
+
+## Guest profile and dedicated user account
+
++ In order to use this feature, you need to create a new user account in Bonita organisation. This account will be the one used to log in automatically a user that tries to access a public living application if he is not logged in to Bonita platform yet.
+
++ You also need to create a new custom profile and add the user account created in the previous step as a member of this profile. This profile will be the one to use as the profile of the living applications that require public access. If an application should also be accessible by users already logged in with their own bonita account, then they should also be members of this profile. 
+
```

Comment:
I don't understand something here:

If i'm correct, this `guest feature` works as follow: 
- A dedicated user is used as the guest user, this user is referenced in the configuration
- Any living application this user can access becomes accessible anonymously 

-> Why is it mandatory to create a custom profile? It doesn't work with the default profiles? 
for example, if my living app is associated to the profile User, and my anonymous account has the profile User, the app won't be accessible anonymously?

Reply:
the app needs to be declared in the conf file below in order for the autologin as guest user to be performed when acessing it.
once you are logged in as guest, it is true that you can access any application this user can access.
It is not mandatory to create a custom profile but if you do not and use the user profile, it means you also guive access to the user profile in the portal to the guest user (just by modifying the URL). Which we don't want.

Reply:
I added a warning to explain that

Reply:
👍 
I guess that the portal doesn't need to be declared in the conf file to gain anonymous access because it is not a living app?

Reply:
giving access to the portal as guest is not something we want to enable. Only living apps